pkg/internalapis/cnsvolumeoperationrequest/cnsvolumeoperationrequest.go

```diff
@@ -286,3 +290,87 @@ func (or *operationRequestStore) StoreRequestDetails(
 	)
 	return nil
 }
+
+// deleteRequestDetails deletes the input CnsVolumeOperationRequest instance from the operationRequestStore.
+func (or *operationRequestStore) deleteRequestDetails(ctx context.Context, name string) error {
+	log := logger.GetLogger(ctx)
+	log.Debugf("Deleting CnsVolumeOperationRequest instance with name %s/%s", csiconfig.DefaultCSINamespace, name)
+	err := or.k8sclient.Delete(ctx, &cnsvolumeoperationrequestv1alpha1.CnsVolumeOperationRequest{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: csiconfig.DefaultCSINamespace,
+		},
+	})
+	if err != nil {
+		if !apierrors.IsNotFound(err) {
+			log.Errorf("failed to delete CnsVolumeOperationRequest instance %s/%s with error: %v",
+				csiconfig.DefaultCSINamespace, name, err)
+			return err
+		}
+	}
+	return nil
+}
+
+// cleanupStaleInstances cleans up CnsVolumeOperationRequest instances for volumes that are no longer present in the
+// kubernetes cluster.
+func (or *operationRequestStore) cleanupStaleInstances(cleanupInterval int) {
+	ticker := time.NewTicker(time.Duration(cleanupInterval) * time.Minute)
+	ctx, log := logger.GetNewContextWithLogger()
+	log.Infof("CnsVolumeOperationRequest clean up interval is set to %d minutes", cleanupInterval)
+	for range ticker.C {
+		instanceMap := make(map[string]bool)
+
```

Comment:
How about one info log with "Deleting CnsVolumeOperationRequest instances". It will log once in 24 hours, so that we can be sure the thread is cleaning up instances. Otherwise I did not see any other info log which will confirm that cleanup is working as expected.

Reply:
done